test/liquid_handle_dispense_test.py

```diff
@@ -856,6 +858,170 @@ def test_tempest_bad_chip_param(self):
+            }
+        }
+
+        liha_shape = instruction.data["shape"]
+        assert liha_shape["rows"] == 1 and liha_shape["columns"] == 1
+        assert instruction.data["mode_params"] == mode_params
+
+    def test_mantis_bad_params(self):
+        """Tests mantis bad params"""
+        # Incorrect model param and incorrect liha shape
+        with pytest.raises(ValueError):
+            self.protocol.liquid_handle_dispense(
+                source=self.tube.well(0),
+                destination=self.flat.well(0),
+                volume="5:uL",
+                liquid=ProteinBuffer,
+                device="x_mantis",
+            )
+        # Incorrect diaphragm value
+        with pytest.raises(ValueError):
+            self.protocol.liquid_handle_dispense(
+                source=self.tube.well(0),
+                destination=self.flat.well(0),
+                volume="5:uL",
+                rows=1,
+                columns=1,
+                liquid=ProteinBuffer,
+                device="x_mantis",
+                diaphragm=101,
+            )
+            # Incorrect nozzle_size value
+            with pytest.raises(ValueError):
+                self.protocol.liquid_handle_dispense(
+                    source=self.tube.well(0),
+                    destination=self.flat.well(0),
+                    volume="5:uL",
+                    rows=1,
+                    columns=1,
+                    liquid=ProteinBuffer,
+                    device="x_mantis",
+                    nozzle_size="0.3:mm",
+                )
+            # Incorrect tubing value
+            with pytest.raises(ValueError):
+                self.protocol.liquid_handle_dispense(
+                    source=self.tube.well(0),
+                    destination=self.flat.well(0),
+                    volume="5:uL",
+                    rows=1,
+                    columns=1,
+                    liquid=ProteinBuffer,
+                    device="x_mantis",
+                    tubing="MV",
+                )
+            # Incorrect z_drop value
+            with pytest.raises(ValueError):
+                self.protocol.liquid_handle_dispense(
+                    source=self.tube.well(0),
+                    destination=self.flat.well(0),
+                    volume="5:uL",
+                    rows=1,
+                    columns=1,
+                    liquid=ProteinBuffer,
+                    device="x_mantis",
+                    z_drop="200.0:mm",
+                )
+            # Incorrect viscosity value
+            with pytest.raises(ValueError):
+                self.protocol.liquid_handle_dispense(
+                    source=self.tube.well(0),
+                    destination=self.flat.well(0),
+                    volume="5:uL",
+                    rows=1,
+                    columns=1,
+                    liquid=ProteinBuffer,
+                    device="x_mantis",
+                    viscosity="100",
+                )
```

Comment:
can you add one more bad test that asserts we can't pass in `x_mantis` with a tempest-specific param?

Reply:
done!